first homework/task2.js

```diff
@@ -1,21 +1,20 @@
 import csv from 'csvtojson';
-import { writeFile } from 'fs';
+import { createWriteStream } from 'fs';
 const csvFilePath='csv\\nodejs-hw1-ex1.csv';
 
-var data='';
+let stream=createWriteStream('test.txt');
 csv()
 .fromFile(csvFilePath)
```

Comment:
method _fromFile_ parses the file and loads it to memory. 
You should use _fromStream_ instead, to make your code be able to work with large files (use _fs.createReadStream_ to prepare stream)

https://stackoverflow.com/questions/16831250/how-to-convert-csv-to-json-in-node-js